.travis.yml

```diff
@@ -12,12 +12,18 @@ matrix:
         - php: 5.5
           dist: trusty
         - php: 5.6
+          dist: xenial
         - php: 7.0
+          dist: xenial
         - php: 7.1
+          dist: bionic
         - php: 7.2
+          dist: bionic
         - php: 7.3
-        - php: 7.4snapshot
-        - php: hhvm
+          dist: bionic
+        - php: 7.4
+          dist: bionic
+        - php: hhvm-3.18
```

Comment:
let's update the `allowed_failures` entry to keep this from making too much noise?

Reply:
Oh yes.